### ISSUE TITLE
Polymer 3 compatible?

### ISSUE THREAD

**Issue by derhuebiii** [greenyouse/bottom-nav#10]
Hi,

I am planning to move to Polymer 3 soon. Is it `bottom-nav`compatible or will it be soon?

Thanks!
Tim

**Comment by greenyouse** [comment]
Good idea! I've been slow on picking up Polymer 3 so I'll try to do some homework and update the project over the next week or two.

**Comment by aletorrado** [comment]
I've just ported this project to polymer 3 for my project. Would you like a PR?

**Comment by greenyouse** [comment]
@aletorrado that sounds great! Sorry I've been slow with upgrading to polymer 3.

**Comment by aletorrado** [comment]
Great. I've just pushed #12 :)
